app/_includes/md/konnect/azure-peering.md

```diff
@@ -0,0 +1,45 @@
+<!--vale off -->
+{% mermaid %}
+flowchart LR
+
+A(API or service)
+B(API or service)
+C(API or service)
+
+G(<img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-height:32px" class="no-image-expand"/>Konnect \n#40;fully-managed \ndata plane#41;)
+H(<img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-height:32px" class="no-image-expand"/>Konnect \n#40;fully-managed \ndata plane#41;)
+J(fa:fa-wifi \n Internet)
+
+subgraph 1 [User Azure Cloud]
+    subgraph 2 [Region]
```

Comment:
I think the Region subgraph threw me off a bit since there was only the one region. What would this look like with multiple regions? I think that would be more helpful to visualize this

Reply:
@Guaris For transit gateways, the user Azure cloud part makes sense, but the Kong AWS cloud part I think could be improved by adding another region to the example. In particular, I'm left wondering if I have two regions (which I think is common with global businesses), which resources live where? The preview link hasn't generated yet, so I'm basing this off the mermaid code, but for the Konnect fully managed data and API or service, would those be shared between regions? Would they only exist in one region? 

Or for transit gateways, if I had two regions, would I have an AWS transit gateway attachment inside each region? Would I have one attachment for all regions?